lib/auth/oidc_test.go

```diff
@@ -80,7 +80,7 @@ func setUpSuite(t *testing.T) *OIDCSuite {
 
 // createInsecureOIDCClient creates an insecure client for testing.
```

Comment:
General comment: I know that testing OIDC is a pain, but we really need tests for this new feature.